hubconf.py

```diff
@@ -1,69 +1,57 @@
-dependencies = ['torch', 'torchaudio', 'numpy', 'vocos', 'safetensors']
```

Comment:
this is a requirement of hubconf.py -- the first line should always be a python list of string pip packages as dependancies. Removing this is probably not a good idea.

Reply:
I see you added it later on -- but I think it must be added before any import calls. Pls double check this